### ISSUE TITLE
Cannot install older versions of icinga2 than v.2.10.0

### ISSUE THREAD

**Issue by miso231** [Icinga/icinga-packaging#116]
I cannot install older versions of icinga than 2.10.0 on Centos7.
```
[root@host ~]# yum install icinga2-2.9.1-1.el7.icinga.x86_64
Loaded plugins: fastestmirror, versionlock
Loading mirror speeds from cached hostfile
 * base: mirror.karneval.cz
 * epel: mirror.karneval.cz
 * extras: mirror.karneval.cz
 * updates: mirror.karneval.cz
Resolving Dependencies
--> Running transaction check
---> Package icinga2.x86_64 0:2.9.1-1.el7.icinga will be installed
--> Processing Dependency: icinga2-bin = 2.9.1-1.el7.icinga for package: icinga2-2.9.1-1.el7.icinga.x86_64
--> Running transaction check
---> Package icinga2-bin.x86_64 0:2.9.1-1.el7.icinga will be installed
--> Processing Dependency: icinga2-libs = 2.9.1-1.el7.icinga for package: icinga2-bin-2.9.1-1.el7.icinga.x86_64
Package icinga2-libs is obsoleted by icinga2-bin, but obsoleting package does not provide for requirements
--> Processing Dependency: libboost_system-mt.so.1.53.0()(64bit) for package: icinga2-bin-2.9.1-1.el7.icinga.x86_64
--> Processing Dependency: libboost_thread-mt.so.1.53.0()(64bit) for package: icinga2-bin-2.9.1-1.el7.icinga.x86_64
--> Processing Dependency: libboost_regex-mt.so.1.53.0()(64bit) for package: icinga2-bin-2.9.1-1.el7.icinga.x86_64
--> Processing Dependency: libboost_program_options-mt.so.1.53.0()(64bit) for package: icinga2-bin-2.9.1-1.el7.icinga.x86_64
--> Running transaction check
---> Package boost-program-options.x86_64 0:1.53.0-27.el7 will be installed
---> Package boost-regex.x86_64 0:1.53.0-27.el7 will be installed
--> Processing Dependency: libicuuc.so.50()(64bit) for package: boost-regex-1.53.0-27.el7.x86_64
--> Processing Dependency: libicui18n.so.50()(64bit) for package: boost-regex-1.53.0-27.el7.x86_64
--> Processing Dependency: libicudata.so.50()(64bit) for package: boost-regex-1.53.0-27.el7.x86_64
---> Package boost-system.x86_64 0:1.53.0-27.el7 will be installed
---> Package boost-thread.x86_64 0:1.53.0-27.el7 will be installed
---> Package icinga2-bin.x86_64 0:2.9.1-1.el7.icinga will be installed
--> Processing Dependency: icinga2-libs = 2.9.1-1.el7.icinga for package: icinga2-bin-2.9.1-1.el7.icinga.x86_64
Package icinga2-libs is obsoleted by icinga2-bin, but obsoleting package does not provide for requirements
--> Running transaction check
---> Package icinga2-bin.x86_64 0:2.9.1-1.el7.icinga will be installed
--> Processing Dependency: icinga2-libs = 2.9.1-1.el7.icinga for package: icinga2-bin-2.9.1-1.el7.icinga.x86_64
Package icinga2-libs is obsoleted by icinga2-bin, but obsoleting package does not provide for requirements
---> Package libicu.x86_64 0:50.1.2-15.el7 will be installed
--> Finished Dependency Resolution
Error: Package: icinga2-bin-2.9.1-1.el7.icinga.x86_64 (icinga2)
           Requires: icinga2-libs = 2.9.1-1.el7.icinga
           Available: icinga2-libs-2.5.0-1.el7.centos.x86_64 (icinga2)
               icinga2-libs = 2.5.0-1.el7.centos
           Available: icinga2-libs-2.5.0-2.el7.centos.x86_64 (icinga2)
               icinga2-libs = 2.5.0-2.el7.centos
           Available: icinga2-libs-2.5.0-3.el7.centos.x86_64 (icinga2)
               icinga2-libs = 2.5.0-3.el7.centos
           Available: icinga2-libs-2.5.1-1.el7.centos.x86_64 (icinga2)
               icinga2-libs = 2.5.1-1.el7.centos
           Available: icinga2-libs-2.5.2-1.el7.centos.x86_64 (icinga2)
               icinga2-libs = 2.5.2-1.el7.centos
           Available: icinga2-libs-2.5.3-1.el7.centos.x86_64 (icinga2)
               icinga2-libs = 2.5.3-1.el7.centos
           Available: icinga2-libs-2.5.4-1.el7.centos.x86_64 (icinga2)
               icinga2-libs = 2.5.4-1.el7.centos
           Available: icinga2-libs-2.6.0-1.el7.centos.x86_64 (icinga2)
               icinga2-libs = 2.6.0-1.el7.centos
           Available: icinga2-libs-2.6.1-1.el7.centos.x86_64 (icinga2)
               icinga2-libs = 2.6.1-1.el7.centos
           Available: icinga2-libs-2.6.2-1.el7.centos.x86_64 (icinga2)
               icinga2-libs = 2.6.2-1.el7.centos
           Available: icinga2-libs-2.6.3-1.el7.centos.x86_64 (icinga2)
               icinga2-libs = 2.6.3-1.el7.centos
           Available: icinga2-libs-2.7.0-1.el7.icinga.x86_64 (icinga2)
               icinga2-libs = 2.7.0-1.el7.icinga
           Available: icinga2-libs-2.7.1-1.el7.icinga.x86_64 (icinga2)
               icinga2-libs = 2.7.1-1.el7.icinga
           Available: icinga2-libs-2.7.2-1.el7.icinga.x86_64 (icinga2)
               icinga2-libs = 2.7.2-1.el7.icinga
           Available: icinga2-libs-2.8.0-1.el7.icinga.x86_64 (icinga2)
               icinga2-libs = 2.8.0-1.el7.icinga
           Available: icinga2-libs-2.8.1-1.el7.icinga.x86_64 (icinga2)
               icinga2-libs = 2.8.1-1.el7.icinga
           Available: icinga2-libs-2.8.2-1.el7.icinga.x86_64 (icinga2)
               icinga2-libs = 2.8.2-1.el7.icinga
           Available: icinga2-libs-2.8.3-1.el7.icinga.x86_64 (icinga2)
               icinga2-libs = 2.8.3-1.el7.icinga
           Available: icinga2-libs-2.8.4-1.el7.icinga.x86_64 (icinga2)
               icinga2-libs = 2.8.4-1.el7.icinga
           Available: icinga2-libs-2.9.0-1.el7.icinga.x86_64 (icinga2)
               icinga2-libs = 2.9.0-1.el7.icinga
           Available: icinga2-libs-2.9.1-1.el7.icinga.x86_64 (icinga2)
               icinga2-libs = 2.9.1-1.el7.icinga
           Available: icinga2-libs-2.9.2-1.el7.icinga.x86_64 (icinga2)
               icinga2-libs = 2.9.2-1.el7.icinga
 You could try using --skip-broken to work around the problem
 You could try running: rpm -Va --nofiles --nodigest
```

**Comment by thedoc31** [comment]
We have the same problem on Amazon Linux 1, trying to install 2.8.4-1.el6.

**Comment by lazyfrosch** [comment]
Same issue as #114 

Please read https://github.com/Icinga/icinga-packaging/issues/114#issuecomment-429264827

TL;DR: It's a problem with yum's dep resolver.
